[docs] Add info about `popToTop` action in Expo Router Stack navigation guide (#24634)
expo/expo: docs/pages/router/advanced/stack.mdx

}
```

## `popToTop` action

> By default, the screens in a `Stack` are pushed or popped to the next or the previous screen as per the order defined within the stack. To go back to the first screen in the existing stack, the [`popToTop`](https://reactnavigation.org/docs/stack-actions/#poptotop) stack action can be used from any screen.

Author: ```suggestion
By default, the screens in a `Stack` are pushed or popped to the next or the previous screen. To return to the first screen in the existing stack, the [`popToTop`](https://reactnavigation.org/docs/stack-actions/#poptotop) stack action can be used from any screen.
```

For example, a stack has three screens &mdash; `home`, `details`, and `settings`. The following layout describes the order of the screens in a stack.

<FileTree files={['app/_layout.js', 'app/home.js', 'app/details.js', 'app/settings.js']} />

The `home` screen is the first screen, and the `settings` is the last. To go from `settings` to `home` screen you'll have to go back to `details`. However, using the `popToTop` action, you can go from `settings` to `home` and dismiss any screen in between.

> It is accessed through [`StackActions`](https://reactnavigation.org/docs/stack-actions) from the React Navigation library and is dispatched using [`navigation`](/router/reference/hooks/#usenavigation) prop.

Author: ```suggestion
It is accessible through [`StackActions`](https://reactnavigation.org/docs/stack-actions) from the React Navigation library and dispatched using [`navigation`](/router/reference/hooks/#usenavigation) prop.
```
